woostarter/patterns/page-about-with-store-information-alternative.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Title: About page with store information 
```

Comment:
This has the same name as the non-alternative version of the pattern:

<img width="353" alt="image" src="https://github.com/user-attachments/assets/2898526a-1569-416c-ace8-bec42617f86f" />

Maybe they should have different names so it's more clear that they're different patterns.